mod.ts

```diff
@@ -3,8 +3,8 @@ import { Sha256, HmacSha256 } from 'https://deno.land/std/hash/sha256.ts'
 const NEWLINE = '\n'
 
 interface GetSignedUrlOptions {
-  bucketName: string
-  objectPath: string
+  path: string
```

Comment:
I understand the motivation, but independent `bucketName` and `objectPath` parameters is less ambiguous.

Reply:
Yes. Having used S3 more I agree that these should be separate.

However, I would suggest to use the AWS naming and call them `bucket` and `key`. This way the parameter names will be the same across the various S3 clients and libraries.

Reply:
Understood, yes your suggestion has parity with the AWS SDK so it's the better option.